aiida/tools/visualization/graph.py

```diff
@@ -27,18 +28,22 @@
 
 __all__ = ('Graph', 'default_link_styles', 'default_node_styles', 'pstate_node_styles', 'default_node_sublabels')
 
+LinkAnnotateType = Literal[None, False, 'label', 'type', 'both']
```

Comment:
Do we need to accept both `None` and `False`? I think either one should suffice and probably just get rid of `None` and then change the default for `annotate_links` in `add_incoming` and `add_outgoing` to `False` instead of `None`. All other functions already have `False` as default.

Reply:
done